bpf/tc/attach.go

```diff
@@ -514,3 +528,22 @@ func (ap *AttachPoint) JumpMapFDMapKey() string {
 func (ap *AttachPoint) IfaceName() string {
 	return ap.Iface
 }
+
+// nolint
+func updateJumpMap(obj *libbpf.Obj, isHost bool) error {
```

Comment:
So this function is setting up the default contents of the jump map?  (Because libbpf doesn't do that for us?)

Reply:
Right. libbpf doesn't load the jump maps with program indices